config-model/src/main/java/com/yahoo/searchdefinition/processing/ImportedFieldsResolver.java

```diff
@@ -52,19 +52,27 @@ private DocumentReference validateDocumentReference(TemporaryImportedField impor
         return reference;
     }
 
-    private SDField validateTargetField(TemporaryImportedField importedField, DocumentReference reference) {
+    private SDField validateTargetField(TemporaryImportedField importedField,
+                                        DocumentReference reference,
+                                        boolean validate) {
         String targetFieldName = importedField.targetFieldName();
         Search targetSearch = reference.targetSearch();
-        if (isImportedField(targetSearch, targetFieldName)) {
-            fail(importedField, targetFieldAsString(targetFieldName, reference) + ": Is an imported field. Not supported");
+        if (validate && isImportedField(targetSearch, targetFieldName)) {
+            fail(importedField, targetFieldAsString(targetFieldName, reference) +
+                                ": Is an imported field. Not supported");
         }
         SDField targetField = targetSearch.getConcreteField(targetFieldName);
         if (targetField == null) {
-            fail(importedField, targetFieldAsString(targetFieldName, reference) + ": Not found");
+            fail(importedField, targetFieldAsString(targetFieldName, reference) +
```

Comment:
Missing a test against validate here?

Reply:
No because if this is null we get a NPE when trying to make the transformation.

Reply:
ok